### ISSUE TITLE
System.Xml.XmlException: Invalid character in the given encoding. Line 1, position 1.

### ISSUE THREAD

**Issue by sharpninja** [unoplatform/uno#8803]
### Current behavior

I get this error trying to compile.  Was working fine, then just stopped working.

```
2>    Target _UnoSourceGenerator:
2>        MSBUILD : error : Generation failed: System.AggregateException: One or more errors occurred. ---> System.InvalidOperationException: Generation failed for Uno.UI.SourceGenerators.XamlGenerator.XamlCodeGenerator. System.AggregateException: One or more errors occurred. ---> System.Xml.XmlException: Invalid character in the given encoding. Line 1, position 1.
2>        MSBUILD : error :    at System.Xml.XmlTextReaderImpl.Throw(Exception e)
2>        MSBUILD : error :    at System.Xml.XmlTextReaderImpl.InvalidCharRecovery(Int32& bytesCount, Int32& charsCount)
2>        MSBUILD : error :    at System.Xml.XmlTextReaderImpl.GetChars(Int32 maxCharsCount)
2>        MSBUILD : error :    at System.Xml.XmlTextReaderImpl.ReadData()
2>        MSBUILD : error :    at System.Xml.XmlTextReaderImpl.SwitchEncoding(Encoding newEncoding)
2>        MSBUILD : error :    at System.Xml.XmlTextReaderImpl.ParseXmlDeclaration(Boolean isTextDecl)
2>        MSBUILD : error :    at System.Xml.XmlTextReaderImpl.Read()
2>        MSBUILD : error :    at System.Xml.XmlLoader.Load(XmlDocument doc, XmlReader reader, Boolean preserveWhitespace)
```

### Expected behavior

Compile successfully.

### How to reproduce it (as minimally and precisely as possible)

1. Clone https://github.com/sharpninja/Activate/commit/bbe159e46a0465e034603615d9633b255bbd46de
1. Open in Visual Studio 2022
1. Rebuild Solution 

### Workaround

None found

### Works on UWP/WinUI

No

### Environment

Windows 11

### NuGet package version(s)

```xml
  <ItemGroup>
    <PackageReference Include="Uno.UI" Version="4.2.6" />
    <PackageReference Include="Uno.UI.RemoteControl" Version="4.2.6" Condition="'$(Configuration)'=='Debug'" />
    <PackageReference Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="4.2.6" />
    <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="5.0.0" />
  </ItemGroup>
  <ItemGroup Condition="'$(TargetFramework)'=='net6.0-android'">
    <PackageReference Include="Xamarin.Google.Android.Material" Version="1.4.0.4" />
    <PackageReference Include="Uno.UniversalImageLoader" Version="1.9.35" />
  </ItemGroup>
  <ItemGroup Condition="'$(TargetFramework)'=='net6.0-ios'">
    <PackageReference Include="Uno.Extensions.Logging.OSLog" Version="1.3.0" />
  </ItemGroup>
```

### Affected platforms

Android, Windows

### IDE

Visual Studio 2022

### IDE version

Version 17.3.0 Preview 1.0

**Comment by jeromelaban** [comment]
Thanks for the report. You have two issues here:

- The first one is that you set the `<LangVersion>` explicitly to 10, you should remove it for now. Would you know how this property got set?
- You set the build type `PRIResource` to the `youtube.png` file, this build type is reserved for resw files. If you want to put files accessible as resources, use the `Content` build type.

I'm keeping this issue opened to enhance the error reporting.

**Comment by sharpninja** [comment]
I got past the compile issue by changing the compile type for all images to Content, but you must include the file in the output, it will not read them as a resource.

**Comment by jeromelaban** [comment]
@sharpninja that last part is an issue that has been fixed in the latest uno 4.3-dev bits. Can you try upgrading?

**Comment by sharpninja** [comment]
> @sharpninja that last part is an issue that has been fixed in the latest uno 4.3-dev bits. Can you try upgrading?

What feed and what version number to look for?

**Comment by jeromelaban** [comment]
Just check the pre-release checkbox in your nuget manager, the `dev` packages for uno are on nuget.org.

**Comment by sharpninja** [comment]
> the `dev` packages for uno are on nuget.org.

Great!  Thank you.

**Comment by sharpninja** [comment]
> @sharpninja that last part is an issue that has been fixed in the latest uno 4.3-dev bits. Can you try upgrading?

OK, upgraded everything to the latest prerelease version.  None of my files marked as `Content` are being embedded in the assembly:

```
assembly.GetManifestResourceNames ()
{string[1]}
    [0]: "Strings.en.Resources.resw.Resources.upri"
```

This for TargetFramework `net6.0-android31.0` which is required to upgrade to the newest packages.

**Comment by jeromelaban** [comment]
_Discussion moved to https://github.com/unoplatform/uno/discussions/8818_
